recurly/__init__.py

```diff
@@ -18,8 +19,8 @@
 
 """
 
-
 __version__ = '2.2.17'
+__python_version__ = '.'.join(map(str, sys.version_info[:3]))
```

Comment:
I would assume releaselevel would always be `final` but it might be interesting to capture. I'm still reading about what serial is exactly.